pkg/yang/modules.go

```diff
@@ -229,6 +236,9 @@ func (ms *Modules) FindModuleByNamespace(ns string) (*Module, error) {
 // FindModuleByPrefix either returns the Module specified by prefix or returns
 // an error.
 func (ms *Modules) FindModuleByPrefix(prefix string) (*Module, error) {
+	ms.mu.Lock()
+	defer ms.mu.Unlock()
+
```

Comment:
This function (along with `byPrefix`) has been removed as of v0.3.0 (a backwards-incompatible version increment). Please use the function `FindModuleByPrefix` instead.

Reply:
Sorry, did you mean to suggest a different FindModule* function ?

Reply:
Yes this method no longer exists in v0.3.0. There is a *function* of the same name in node.go